checkstyle-tester/diff.groovy

```diff
@@ -246,31 +249,26 @@ def launchCheckstyleReport(cfg) {
     return reportInfo
 }
 
-def parseProjects(listOfProjectsFile, allowExcludes) {
+def parseProjects(listOfProjectsFile, allowExcludes, includedProjects) {
     def projects = []
-    def fullParamListSize = 5
-
-    new File(listOfProjectsFile).eachLine { project ->
```

Comment:
do not remove old functionality
please do check that file is with `yaml` extension and load it in yaml way, ELSE load it in "property"/old way